client/elements/sc-page-selector.html

```diff
@@ -99,6 +103,10 @@
         background-color: var(--sc-primary-color);
       }
 
+      .logo-hidden {
```

Comment:
A good naming convention for these classes with only one CSS property is to name them more generally by that property: eg. `display-none`, without referring to any specific element (`logo` in this case). This has 2 upsides:
- Looking at the HTML, you know right away what the `display-none` class does,
- If, in the future, you decide to use `display: none` for another node in this file which isn't a logo, you won't have to rename this CSS class.